ocdata/core/multi_adsorbate_slab_config.py

```diff
@@ -0,0 +1,192 @@
+from typing import List
+
+import numpy as np
+from ase import Atoms
+from ase.data import covalent_radii
+
+from ocdata.core import Adsorbate, AdsorbateSlabConfig, Slab
+
+
+class MultipleAdsorbateSlabConfig(AdsorbateSlabConfig):
+    """
+    Class to represent a slab with multiple adsorbates on it. This class only
+    returns a fixed combination of adsorbates placed on the surface. Unlike
+    AdsorbateSlabConfig which enumerates all possible adsorbate placements, this
+    problem gets combinatorially large.
+    """
+
+    def __init__(
+        self,
+        slab: Slab,
+        adsorbates: List[Adsorbate],
+        num_sites: int = 100,
+        num_configurations: int = 1,
+        interstitial_gap: float = 0.1,
+        mode: str = "random_site_heuristic_placement",
```

Comment:
Could you document all of these arguments please (like in the parent class).
- We should note that interstitial_gap is being used both for controlling the min. distance along the surface normal while placing individual adsorbates as well as the inter-adsorbate distance.
- And that num_configurations (also used as an arg to place_adsorbate_on_sites) is different from the `num_augmentations_per_site` argument the parent class specifies for `place_adsorbate_on_sites`.